src/lib.rs

```diff
@@ -70,11 +70,11 @@ impl<R: Read> Decoder<R> {
         let mut pows = [0f32; NELLY_BUF_LEN];
         {
             let mut reader = BitReader::endian(Cursor::new(&block), LittleEndian);
-            let mut val = NELLY_INIT_TABLE[reader.read::<u8>(6).unwrap() as usize] as f32;
+            let mut val = NELLY_INIT_TABLE[reader.read::<6, u8>().unwrap() as usize] as f32;
```

Comment:
According to my understanding (and the mentioned commit) we should be using `read_var` here, unless I'm missing something?

Reply:
Why? They number of bits to be read is constant...?

Reply:
Okay, I assumed there's a difference in behavior (the old `read` method was renamed to `read_var`). So in addition to upgrading we're switching to the version of methods with constants, which looks good!